src/pytorch_lightning/plugins/environments/slurm_environment.py

```diff
@@ -30,9 +30,10 @@ class SLURMEnvironment(ClusterEnvironment):
             rescheduled gets determined by the owner of this plugin.
     """
 
-    def __init__(self, auto_requeue: bool = True) -> None:
+    def __init__(self, auto_requeue: bool = True, signal: str = "USR1") -> None:
```

Comment:
Some considerations:
- We could support int here, i.e., signal: int = signal.SIGUSR1
- We could support both, i.e., signal: int | str = signal.SIGUSR1
- Is excluding the SIG prefix intuitive? Intuitively, I would have passed in `signal.SIGUSR1` or `"SIGUSR1"`. 

No strong opinions here, maybe I'm overthinking this.

Reply:
I think going straight for `signal: int = signal.SIGUSR1` makes sense actually, and this simplifies the code quite a bit so less room for error, I'll make that change. I don't see any particular advantage for the string parameter.